auth-server/src/api/securityQuestionForUser.js

```diff
@@ -5,7 +5,7 @@ const User = require("../models/User");
 const { isSecurityQuestionEnabled } = require("../utils/getConfigFile");
 
 router.post(
-  "/securityQuestionForUser",
```

Comment:
I actually prefer the original route, "get" doesn't follow api naming conventions. Any reason for the change?

Reply:
the forgot password functionality wasn't working cuz there was some mixup with the security question routing/naming so brandon changed them all to be the same, i can rename them all to /securityQuestionForUser if that follows naming conventions more